ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Runner.scala

```diff
@@ -122,8 +122,31 @@ final class Runner[T <: ReadWriteService, Extra](
                     .map(_.start(config.metricsReportingInterval.getSeconds, TimeUnit.SECONDS))
                     .acquire()
                 )
+                servicesExecutionContext <- ResourceOwner
+                  .forExecutorService(() =>
+                    new InstrumentedExecutorService(
+                      Executors.newWorkStealingPool(),
+                      metrics.registry,
+                      metrics.daml.lapi.threadpool.apiServices.toString,
+                    )
+                  )
+                  .map(ExecutionContext.fromExecutorService)
+                  .acquire()
+                apiServerConfig = factory.apiServerConfig(participantConfig, config)
+                indexService <- StandaloneIndexService(
+                  ledgerId = config.ledgerId,
+                  config = apiServerConfig,
+                  metrics = metrics,
+                  engine = sharedEngine,
+                  servicesExecutionContext = servicesExecutionContext,
+                  lfValueTranslationCache = lfValueTranslationCache,
+                ).acquire()
                 ledger <- factory
-                  .readWriteServiceOwner(config, participantConfig, sharedEngine)
+                  .readWriteServiceOwner(config, participantConfig, sharedEngine, indexService)(
+                    materializer,
+                    servicesExecutionContext,
```

Comment:
The issue here it's not that we have too many implicits in scope. We have to be explicit to avoid having to refactor this method so that we can make `servicesExecutionContext` implicit.

Reply:
I understand; I'm asking why it's not an explicit parameter of the `readWriteServiceOwner` method (and other methods lower down).